tests/framework/PostProcessors/ETimporterPostProcessor/test_ETimporterSymbolic.xml

```diff
@@ -0,0 +1,54 @@
+<Simulation verbosity="debug">
+  <TestInfo>
+    <name>framework/PostProcessors/ETimporterPostProcessor</name>
```

Comment:
same as above.

In addition all the <name> nodes you create in the tests have the same name...

Remeber that this names are used by the automatic regression test documentation latex script